tests/mock/performance_mocks.py

```diff
@@ -499,7 +500,7 @@ def __getitem__(self, route_name):
 
     def actual_rest_app(self):
         if self._actual_rest_app is None:
-            self._actual_rest_app, _datastore = make_rest_app(db_filepath="no datastore",
```

Comment:
Good find - this was leaving an artifact in the filesystem after tests; Nonetheless, let's make sure we clean this up too and not just sweep it under the rug.